rev_news/drafts/edition-14.md

```diff
@@ -117,65 +116,65 @@ He is the maintainer of the Kernel '-stable' branches and of many
+in the core, 10% in the networking subsystem, and 55% in the drivers.
 
-This goes against any previously thought methodology for stable
-software. And things are going faster and faster.
+This goes against all previous methodologies for stable
+software development, and things are only getting faster and faster.
 
 Things are going so fast that it costs money to keep your code outside
 the kernel.
 
-There is a new release every 2.5 months, so if your code get rejected
+There is a new release every 2.5 months, so if your code is rejected
 you have to wait 2 months before it can be in the next release. This
 is very predictable.
 
 The release cycle is made of a two week long "merge window" and then
-some "rc" releases one per week. During the "merge window", code is
+some "rc" releases - one per week. During the "merge window", code is
 merged from subsystem maintainers. The "rc" releases, "rc-1", "rc-2",
 ... , "rc-7" are bug fixes only. Once all major bugs and regressions
-are fixed a release is made and the cyce starts over with a ne merge
+are fixed a release is made and the cycle starts over with a new merge
 window.
 
 For "stable" kernels that Greg maintains, they are forked from Linux
 releases. Commits have to go in Linus' tree first before Greg will
-accept them. It should be the identical patch as what is in Linus'
-tree and it should be bug fixes only or new device ids. The Linux
+accept them. It should be the same identical patch as what is in Linus'
+tree, and it should only contain bug fixes or new device ids. The Linux
 distributions usually run from "stable" kernels.
 
-The "longterm" kernel is maintained for 2 years. Current "longterm"
+The "longterm" kernels are maintained for 2 years. Current "longterm"
 kernels are 3.14, 4.1 and 4.4.
 
-This works well for many companies. But sometimes maintaining a kernel
+This works well for many companies, but sometimes maintaining a kernel
 for 2 years is not enough. Japan is converting all its infrastructure
-to Linux and people want some kernels maintained for 20 years.
+to Linux, and people want some kernels maintained for 20 years.
 
 The patches sent for inclusion into the Kernel should be standalone.
-None of them should break the build. One half to one third of the
-patches get accepted. They should be obvious, broken down in the
-smallest possible patches and every change should be correct.
+None of them should break the build. Each patch set should be obvious,
+broken down into the smallest reasonable patches, and every change
+should be correct. One half to one third of submitted patches get accepted. 
 
-This puts work on the developer's shoulders. But that's on purpose
-because there are much more developers than maintainers. There are
-1000 maintainers but only around 700 are active.
+This puts most of the work on the developer's shoulders. That's on purpose,
+as there are many more developers than maintainers. There are
```

Comment:
Ok for s/much/many/ but otherwise I'd rather keep a version that is closer to what Greg said.